codec-http2/src/main/java/io/netty/handler/codec/http2/StreamBufferingEncoder.java

```diff
@@ -72,9 +72,9 @@ public Http2ChannelClosedException() {
         private static final long serialVersionUID = 1326785622777291198L;
         private final int lastStreamId;
         private final long errorCode;
-        private final ByteString debugData;
+        private final AsciiString debugData;
```

Comment:
`byte[]` works for me.

We can impose a limit on the amount we copy with a followup PR if necessary.